packages/flow-table/src/components/f-table-schema/f-table-schema.ts

```diff
@@ -361,6 +361,21 @@ export class FTableSchema extends FRoot {
 		this.searchScope = event.detail.scope;
 		this.searchTerm = event.detail.value;
 	}
+	get noDataTemplate() {
+		if (this.data.rows.length === 0 && this.data.header) {
+			return html`<f-div
+				style="grid-column-start:1;grid-column-end:${Object.keys(this.data.header).length + 1};"
+			>
+				<slot name="no-data">
+					<f-div padding="small" state="secondary" align="middle-center" width="100%">
+						<f-text inline>No data to display</f-text>
```

Comment:
Why hardcoded strings? Should be configurable atleast

Reply:
That's the default message in slot `<slot name="no-data">` if `no-data` slot is not specified.